pages/common/hadolint.md

```diff
@@ -0,0 +1,20 @@
+# hadolint
+
+> Dockerfile linter.
+> More information: <https://github.com/hadolint/hadolint>.
+
+- Lint a Dockerfile:
+
+`hadolint {{path/to/Dockerfile}}`
+
+- Lint a Dockerfile outputting a specific format:
+
+`hadolint --format {{json}} {{path/to/Dockerfile}}`
+
+- Lint a Dockerfile ignoring specific rules:
+
+`hadolint --ignore {{DL3006}} --ignore {{DL3008}} {{path/to/Dockerfile}}`
+
+- Lint multiple Dockerfiles while specifying a trusted registries:
```

Comment:
there are 2 registries. Maybe this is better:
```suggestion
- Lint multiple Dockerfiles using specific trusted registries:
```

Reply:
Agreed. Made the change